calico_cni/policy_drivers.py

```diff
@@ -131,6 +131,14 @@ def generate_tags(self):
         """
         return []
 
+class MesosPolicyDriver(DefaultPolicyDriver):
+    def generate_rules(self):
+        allow = Rule(action="allow")
+        rules = Rules(id=self.profile_name,
+                      inbound_rules=[allow],
```

Comment:
I don't think this is correct - Mesos supports multiple CNI networks that need to be isolated from each other by default, so they shouldn't allow all inbound by default.